recipes-core/images/iot2050-image-example.bb

```diff
@@ -41,6 +41,9 @@ IMAGE_INSTALL += " \
     libteec1 \
     optee-client-dev \
     tee-supplicant \
+    iot2050-eio-manager \
+    iot2050-conf-webui \
+    iot2050-event-record \
```

Comment:
This should become a config option, maybe default-y but not unconditionally included - may too heavy.

Reply:
separated from the main config, and added Kconfig entry.